bedrock/firefox/templates/firefox/switch.html

```diff
@@ -0,0 +1,87 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/base-pebbles.html" %}
+
+{% block page_title %}{{ _('Switch from Chrome to Firefox in just a few minutes') }}{% endblock %}
+{% block page_desc %}{{ _('Switching to Firefox is fast, easy and risk-free, because Firefox imports your bookmarks, autofills, passwords and preferences from Chrome.') }}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox-switch' %}
+{% endblock %}
+
+{% block body_id %}firefox-switch{% endblock %}
+
+{% block optimizely %}
+  {% if switch('firefox-switch-optimizely', ['en-US']) %}
+    {% include 'includes/optimizely.html' %}
+  {% endif %}
+{% endblock %}
+
+{% block content %}
+{% include 'firefox/includes/hub/sub-nav.html' %}
+<main>
+  <div class="content">
+    <div class="inner-container">
+      <header>
+        <h1>{{ _('Switch from Chrome to Firefox in just a few minutes') }}</h1>
+        <p class="tagline">{{ _('Switching to Firefox is fast, easy and risk-free. Firefox imports your bookmarks, autofills, passwords and preferences from Chrome.') }}</p>
+      </header>
+
+      <ol class="prose">
+        <li>{{ _('Select what to take from Chrome.') }}</li>
+        <li>{{ _('Let Firefox do the rest.') }}</li>
+        <li>{{ _('Enjoy the web faster, all set up for you.') }}</li>
+      </ol>
+
+      {{ download_firefox(alt_copy=_('Download and switch'), download_location='primary cta') }}
+
+      <footer>
+        <ul>
+          <li>
+            {{ _('You can use Firefox and still have Chrome. Chrome won’t change on your machine one bit.') }}
+          </li>
+
+          <li>
+              <h2>{{ _('Share with your friends how to switch to Firefox') }}</h2>
+
+              {% set share_url = settings.CANONICAL_URL + canonical_path %}
+
+              <ul class="share-options">
+                <li>
+                  <a data-link-type="link" data-link-name="Share to Facebook" href="https://www.facebook.com/sharer/sharer.php?s=100&p[url]={{ share_url|urlencode }}">
+                    {{ _('Share to Facebook') }}
+                  </a>
+                </li>
+                <li>
+                  {# L10n: the string below is for a referral email subject line. #}
+                  {% set email_subject = _('Switch to Firefox')|urlencode %}
+
+                  {# L10n: the string below is for a referral email message body. Use \n\n for line breaks in the message as appropriate. At the end of the string after the colon, a URL will be added automatically. #}
+                  {% set email_body = _('Hey, \n\nFirefox makes switching from Chrome really fast. I like it a lot, and you should try it. \n\nCheck it out and let me know what you think: ')|urlencode %}
```

Comment:
There are two issues here:
* The link doesn't really work, it creates an email with + instead of sèaces
* The main problem is that `\n` gets lost in the string extraction. It becomes
```
Hey, Firefox makes switching from Chrome really fast. I like it a lot, and you should try it. Check it out and let me know what you think: 
```

And it gets correctly read from the lang file (i.e. it shows up localized if I change it), which means `\n` is not only stripped by the extraction process, but also ignored by Django.

I think we need `<br>` here, or split the string in two parts. Note that this is blocking extracting strings.

Reply:
One more thing that I forgot: we don't support leading or trailing spaces in strings, they're stripped off.